CHANGELOG.md

```diff
@@ -11,7 +11,9 @@ Changelog for package power_msgs
 * Merge pull request `#6 <https://github.com/fetchrobotics/power_msgs/issues/6>`_ from macmason/master
   Add LICENSE file.
 * Add LICENSE file.
-* Contributors: Carl Saldanha, Mac Mason, Michael Ferguson, Russell Toris, cjds
+* Add total_capacity, current_capacity, battery_voltage, supply_voltage and charger voltage to battery
+ state
+* Contributors: Carl Saldanha, Mac Mason, Michael Ferguson, Pavan Soundara, Russell Toris
```

Comment:
@pavansoundara this changelog should be auto-generated with catkin_generate_changelog."
it should remain CHANGELOG.rst

http://wiki.ros.org/bloom/Tutorials/ReleaseCatkinPackage